app/views/landing.html

```diff
@@ -73,6 +73,10 @@ <h1 class="heading">Scenic Subscription offer</h1>
       // Start subscription flow with the target SKU.
       subscriptions.subscribe('subscription_daily');
     });
+    if ('<% logImpressions %>') {
```

Comment:
I think we need to skip `if`. And I also want to make it canonical to always have `.configure()` calls before calls to `.init()`.